### PR TITLE
Place description

### LINKED ISSUE #4: Place description
Screenshot 2019-10-22 at 20 06 35

### LINKED ISSUE #5: Opening hours
Screenshot 2019-10-22 at 20 06 38

### PR DESCRIPTION
**Pull request by qwercik**
Features:
 - Configured basic routing
 - Add restaurant info page
 - Add components with place description and working hours
I know there are still few things to change or improve. I'm still working on them. But you can take a look and express your general opinion.

Closes #4 
Closes #5

### CODE REVIEW

File: src/components/BackButton/BackButton.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.back-button {
+  position: absolute;
+  top: 20px;
+  left: 20px;
+  border: 1px solid #DDDDDD;
```
Comment (roertbb) [inline src/components/BackButton/BackButton.scss]: do we need that border?

File: src/components/OpeningHours/OpeningHours.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+          <div className="day-name">
+            Niedziela
+          </div>
+          <div className="hours-interval">
+            12:00 - 24:00
+          </div>
+        </li>
+      </ol>
+
+      <div class="closing-soon-info">
```
Comment (roertbb) [inline src/components/OpeningHours/OpeningHours.js]: `class` -> `className` as above 😉

File: src/components/PlaceDescription/PlaceDescription.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.content {
+  color: #32313c;
```
Comment (roertbb) [inline src/components/PlaceDescription/PlaceDescription.scss]: what about using sass variables for color? @ryszard-put should have already extracted some

File: src/pages/MapPage.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
```
Comment (roertbb) [inline src/pages/MapPage.js:1]: 👍 for spliting into `pages`

File: src/pages/RestaurantPageInfo/RestaurantPageInfo.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PlaceDescription from '../../components/PlaceDescription/PlaceDescription';
+import OpeningHours from '../../components/OpeningHours/OpeningHours';
+import RestaurantPicture from '../../assets/images/restaurant.jpg';
+import './RestaurantPageInfo.scss';
+import BackButton from '../../components/BackButton/BackButton';
+
+export default function RestaurantInfoPage() {
+  return (
+    <div className="RestaurantPageInfo">
```
Comment (roertbb) [inline src/pages/RestaurantPageInfo/RestaurantPageInfo.js]: can we use lowercase, hyphen-separated class here also?

File: src/pages/RestaurantPageInfo/RestaurantPageInfo.scss [hunk 6]
```diff
@@ -0,0 +1,19 @@
+.restaurant-picture {
+  width: 100%;
+  z-index: -1;
+}
+
+.restaurant-info-content {
+  background-color: white;
+  padding: 40px;
+  border-radius: 20px 20px 0 0;
+  border: 1px solid #BBBBBB;
```
Comment (roertbb) [inline src/pages/RestaurantPageInfo/RestaurantPageInfo.scss]: do we need that border here?

File: src/pages/RestaurantPageInfo/RestaurantPageInfo.scss [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+  position: relative;
+  top: -25px;
+
+  min-height: 100vh;
```
Comment (roertbb) [inline src/pages/RestaurantPageInfo/RestaurantPageInfo.scss]: what if we get rid of that `min-height` and let that container's content define it's size - just as we've discussed?

**Review by roertbb — COMMENTED** [review]
Good job 👍 kudos for introducing router 😉

File: src/components/BackButton/BackButton.scss (hunk 1 above)
Comment (qwercik) [reply src/components/BackButton/BackButton.scss]: No, it wasn't necessary

File: src/components/OpeningHours/OpeningHours.js (hunk 2 above)
Comment (qwercik) [reply src/components/OpeningHours/OpeningHours.js]: Oh, I still can't get used to React :smiley:

File: src/components/PlaceDescription/PlaceDescription.scss (hunk 3 above)
Comment (qwercik) [reply src/components/PlaceDescription/PlaceDescription.scss]: Yes, it's of course a good idea. I replaced my colors' values with existing SASS variables.

File: src/pages/RestaurantPageInfo/RestaurantPageInfo.js (hunk 5 above)
Comment (qwercik) [reply src/pages/RestaurantPageInfo/RestaurantPageInfo.js]: Oh, that is an effect of writing code at night or in the late evening. Thank You :smile:

File: src/pages/RestaurantPageInfo/RestaurantPageInfo.scss (hunk 6 above)
Comment (qwercik) [reply src/pages/RestaurantPageInfo/RestaurantPageInfo.scss]: No, it's also not necessary.

File: src/pages/RestaurantPageInfo/RestaurantPageInfo.scss (hunk 7 above)
Comment (qwercik) [reply src/pages/RestaurantPageInfo/RestaurantPageInfo.scss]: I replaced min-height with an margin-bottom. Could you take a look at this?

File: src/pages/MapPage.js (hunk 4 above)
Comment (qwercik) [reply src/pages/MapPage.js:1]: Thank you

**Pull request merged**